gut_to_soil_manuscript_figures/scripts/plot_pcoa_2d.py

```diff
@@ -475,9 +484,16 @@ def plot_pcoa_2d(metadata_fp, ordination_fp, measure,
             ax.plot([x0_bulk_mean, x1_mean], [y0_bulk_mean, y1_mean],
                     '--', color='#C5C9C7', linewidth=0.75, zorder=2)
 
+    # handling title text for buckets depending on bucket highlighting
```

Comment:
hey @gregcaporaso i added this segment below to improve the title text - i'll attach a few images to the main PR description so you can see what this looks like. other than this update, everything looks g2g!